apps/services/auth-api/esbuild.json

```diff
@@ -26,7 +26,21 @@
     "pg-native",
     "form-data",
     "configcat-node",
-    "sequelize"
+    "sequelize",
+    "@theo.gravity/datadog-apm",
+    "@datadog/native-metrics",
+    "@datadog/pprof",
+    "@datadog/sketches-js",
+    "@protobufjs/aspromise",
+    "@protobufjs/base64",
+    "@protobufjs/codegen",
+    "@protobufjs/eventemitter",
+    "@protobufjs/fetch",
+    "@protobufjs/float",
+    "@protobufjs/inquire",
+    "@protobufjs/path",
+    "@protobufjs/pool",
+    "@protobufjs/utf8"
```

Comment:
This is annoying :(

Reply:
Yeah, this was necessary for the tracing. Am I right @pshomov?